classes/CurseTwitter.php

```diff
@@ -79,51 +67,39 @@ public function setHeight($height) {
 
 	/**
 	 * Background Theme Setting
-	 *
-	 * @access	public
-	 * @param	array	Background Theme
-	 * @return	string	$theme
 	 */
-	public function setTheme($background) {
+	public function setTheme($background): void
+    {
 		if ($background) {
 			$this->attributes['data-theme'] = $background;
 		}
 	}
 
 	/**
 	 * Set link color.
-	 *
-	 * @access	public
-	 * @param	array	Link Color
-	 * @return	string	$links
 	 */
-	public function setLinkColor($linkColor) {
+	public function setLinkColor($linkColor): void
+    {
 		if ($linkColor) {
 			$this->attributes['data-link-color'] = $linkColor;
 		}
 	}
 
 	/**
 	 * Set border color.
-	 *
-	 * @access	public
-	 * @param	array	Border Color
-	 * @return	string	$borders
 	 */
-	public function setBorderColor($borderColor) {
+	public function setBorderColor($borderColor): void
+    {
 		if ($borderColor) {
 			$this->attributes['data-border-color'] = $borderColor;
 		}
 	}
 
 	/**
 	 * Check settings for chrome elements.
-	 *
-	 * @access	public
-	 * @param	array	args
-	 * @return	string	$chrome
 	 */
-	public function setChrome(&$args) {
+	public function setChrome(array &$args): void
```

Comment:
```suggestion
	public function setChrome(array $args): void
```
no need for reference, we're not changing it here